### PR TITLE
Add Mapzen JS docs

### PR DESCRIPTION
**Pull request by migurski**
Adds a new Maps section for Mapzen JS API documentation.

* [x] Wait for https://github.com/mapzen/mapzen.js/tree/kkowalsky-text-edit/docs to be completed.

/cc @kkowalsky.

### CODE REVIEW

**Comment by rmglennon** [comment]
Let's make the description on in the docs landing page block and on the doc page start with a verb to match the others: 
- Everything you need to use Mapzen services in your web applications. 
- A thin wrapper that packages up everything you need to use Mapzen services in your web applications.

Even if it's "Get everything you need..."

File: config/mapzen-js.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+site_name: Mapzen Javascript SDK
```
Comment (rmglennon) [inline config/mapzen-js.yml]: Let's make it JavaScript, with upper S. 

Or perhaps it's just Mapzen.js (styled however) with the words JavaScript SDK in the text underneath?

**Comment by kkowalsky** [comment]
For now we have the scarab on the docs page, but does it make sense to include it on the website?

**Comment by migurski** [comment]
Yes, it makes sense to include the Scarab as long as the primary users are internal. We can de-emphasize it over time, but for now let’s keep it.

**Comment by rmglennon** [comment]
See PR https://github.com/mapzen/mapzen.js/pull/193, which will add these to master over in mapzen.js repo. Should be ready to go.

**Pull request merged**